problems/1.types+operators.js

```diff
@@ -1,21 +1,23 @@
 /** Takes in an value and returns if it is a number
-* @param {any} val - The input value
-* @returns {boolean} - Is val a number
-*/
-function isNumber() {
-
-}
+ * @param {any} val - The input value
+ * @returns {boolean} - Is val a number
+ */
+function isNumber(num) {
+  let age = num
+    return age +7
```

Comment:
Why age + 7 ? 
We need to find out if num is a number we can use typeof to do this.